lib/verify.js

```diff
@@ -166,7 +166,15 @@ export default async function verify(pluginConfig, context, { Octokit }) {
       status,
       data: { clone_url },
     } = await octokit.request("GET /repos/{owner}/{repo}", { owner, repo });
-    if (status !== 200 || repositoryUrl !== clone_url) {
+
+    // compare only the <owner>/<repo> part of the two URLs in case one URL is HTTPS
+    // and the other is SSH
+    const getRepoPath = url => new URL(url).pathname.replace(/\.git$/, '')
```

Comment:
We already have a utility function that you can use in place of this `parseGitHubUrl`... see https://github.com/semantic-release/github/blob/master/lib/parse-github-url.js